.pre-commit-config.yaml

```diff
@@ -29,6 +29,7 @@ repos:
     rev: v2.2.1
     hooks:
       - id: codespell
+        args: ["-L", "lod"]
```

Comment:
We can add the word "load" at https://github.com/open-mmlab/mmeval/blob/main/setup.cfg#L21 rather than here.